infra/lib/api-stack/fhir-converter-connector.ts

```diff
@@ -0,0 +1,143 @@
+import { Duration } from "aws-cdk-lib";
+import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import { IVpc } from "aws-cdk-lib/aws-ec2";
+import { Function as Lambda } from "aws-cdk-lib/aws-lambda";
+import { SqsEventSource } from "aws-cdk-lib/aws-lambda-event-sources";
+import * as s3 from "aws-cdk-lib/aws-s3";
+import { DeadLetterQueue, Queue } from "aws-cdk-lib/aws-sqs";
+import { Construct } from "constructs";
+import { EnvType } from "../env-type";
+import { settings as settingsFhirConverter } from "../fhir-converter-service";
+import { getConfig, METRICS_NAMESPACE } from "../shared/config";
+import { createLambda as defaultCreateLambda } from "../shared/lambda";
+import { createQueue as defaultCreateQueue, provideAccessToQueue } from "../shared/sqs";
+
+function settings() {
+  const { cpuAmount: fhirConverterCPUAmount, taskCountMin: fhirConverterTaskCounMin } =
+    settingsFhirConverter();
+  const lambdaTimeoutSeconds = 10 * 60;
```

Comment:
NBD now, but at some point using a lib for time duration conversions would be a good precedence - we hardcode s/m/ms conversions in a variety of places

Reply:
Agreed! Also for types: UUID, ISO8601, etc., those could be types as well.

For durations, `dayjs` has a plugin: https://day.js.org/docs/en/durations/milliseconds

https://github.com/metriport/metriport-internal/issues/729